### ISSUE TITLE
Is Rust+Flutter ok for real-world production app?

### ISSUE THREAD

**Issue by fzyzcjy** [shekohex/flutterust#30]
Hi thanks for this template! I wonder whether Rust+Flutter is ok for real-world production app? Or it is only suitable for toy projects?

**Comment by shekohex** [comment]
Yes, there are more and more projects out there is building on top of Rust and Flutter:

### Under Active Development:
1. [Owlchat](https://github.com/owlchat)
2. [Subsocial Flutter SDK](https://github.com/dappforce/subsocial-flutter)
3. [allo-isolate](https://github.com/sunshine-protocol/allo-isolate) is being used a lot! see the stats from [crates.io](https://crates.io/crates/allo-isolate) and this recent [PR](https://github.com/sunshine-protocol/allo-isolate/pull/3) that introduced zero-copy buffers between Rust <-> Dart! and the comments show that people are using it for serious work.
4. The [RID](https://thlorenz.com/rid-site/) _(Rust Integrated Dart)_ project which is super cool, I got to e-meet with the author of the project @thlorenz at Flying High with Flutter podcast ([link](https://www.youtube.com/watch?v=SsZUicpnpVE))
5. [audiobookfinder](https://github.com/electricherd/audiobookfinder).

You could search more on web about "flutterust" or "flutter and rust" you will find more!

I will leave this issue open if anyone wants to add their projects.

**Comment by fzyzcjy** [comment]
Thank you!
